GeoLib/AABB.h

```diff
@@ -160,9 +181,33 @@ class AABB
 	{
 		init(*pnt);
 	}
+
+	/// Updates the bounding box using point \f$p\f$ without resizing it.
+	std::pair<bool,std::bitset<3>> update_(PNT_TYPE const & p)
```

Comment:
Would you please elaborate more on what update without resizing means?
Maybe return value explanation would already help.